### PR TITLE
Fix data model

### PR DESCRIPTION
**Pull request by dlongley**

### CODE REVIEW

File: test/mocha/30-event-api.js [hunk 1]
```diff
@@ -12,28 +12,27 @@ const mockData = require('./mock.data');
 const uuid = require('uuid/v4');
 
 const exampleLedgerId = 'did:v1:' + uuid.v4();
-const configEventTemplate = mockData.events.config;
-const configBlockTemplate = mockData.configBlocks.alpha;
+const configEventTemplate = _.cloneDeep(mockData.events.config);
+configEventTemplate.ledger = exampleLedgerId;
 
+const configBlockTemplate = _.cloneDeep(mockData.configBlocks.alpha);
```
Comment (mattcollier) [inline test/mocha/30-event-api.js]: I've seen `_.cloneDeep` creeping into code.  Does this do something different/better that `bedrock.util.clone`?

Comment (dlongley) [reply test/mocha/30-event-api.js]: Copied from elsewhere in the test suite in this case. We should probably just start phasing out `bedrock.util.clone` so we don't have to maintain or try and improve the clone method.

Comment (mattcollier) [reply test/mocha/30-event-api.js]: bedrock already has a `lodash` dep, should we just alias `bedrock.util.clone` to `_.cloneDeep`?

Comment (dlongley) [reply test/mocha/30-event-api.js]: That's step one, yeah. We should make sure there's nothing "better" or "special" about what `bedrock.util.clone` is doing first though (I doubt there is anything).

**Pull request merged**